ParisBNF/abb/BNFabb107.xml

```diff
@@ -29,10 +29,214 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            <title ref="LIT5927PrayerBE"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እግዚአብሔ</hi>ር፡ አምላከ፡ እስራኤል፡ አ<hi rend="rubric">ምላከ፡ ለኵሉ፡ ዘነፍስ፡</hi> ወለኵሉ፡ ዘሥጋ፡ ወዝንቱ፡ ውእቱ፡ ስምየ፡ ዘነገርክዎ፡ ለርኂብ፡ ዘይነብብ፡ አፉየ፡
+                        </incipit>
+                            <explicit xml:lang="gez">ብፁዕ፡ ውእቱ፡ ዘተናዘዘ፡ ለዛቲ፡ ጸሎት፡ ዘይቤ፡ መኑ፡ ዝንቱ፡ ዘኢይሄሊ፡ በልቡ፡ ወኢይክሉ፡ ገቢረ፡ ኑፋቄ፡ የዓሥዮ፡ እግዚአብሔር።</explicit>
+                            <listBibl type="text">
+                                <bibl>
+                                    <ptr target="bm:Aescoly1951Recueil"/>
+                                    <citedRange unit="page">176-185</citedRange>
+                                </bibl>
+                            </listBibl>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <locus from="9rb" to="10ra"/>
+                            <title ref="LIT5928IncantationBE"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እግዚአብሔ</hi>ር፡ አምላከ፡ እስራኤል፡ ኤ<hi rend="rubric">ል፡ ኤል፡ ኤል፡ ኤል፡ ኤል፡</hi> ኤል፡ ኤል፡ ኪናኪር፡  
+                            </incipit>
+                            <explicit xml:lang="gez">ወለኵሎሙ፡ መናፍስት፡ ርኩሳን፡ ከመ፡ ኢይቅረቡ፡ ኀበ፡ ነፍሱ፡ ወሥጋሁ፡ ለገብርከ። <hi rend="rubric">ባይገባ፡</hi> </explicit>
+                            <listBibl type="text">
+                                <bibl>
+                                    <ptr target="bm:Aescoly1951Recueil"/>
+                                    <citedRange unit="page">192-193</citedRange>
+                                </bibl>
+                            </listBibl>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <locus target="#10rava"/>
+                            <title ref="LIT5929AsmatBE"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እግዚአብሔ</hi>ር፡ አምላከ፡ እስራኤል፡ <hi rend="rubric">ወፀውኡ፡ ስምየ፡ በሳብ</hi>ዕት፡ ዕለት፡ ወበሳብዕት፡ ሰንበት፡ ኤሌኤል፡  
+                            </incipit>
+                            <explicit xml:lang="gez">ወስቡሕ፡ ይትአኰት፡ ስሙ፡  ለእግዚአብሔር፡ ሃሌሉያ፡ ተአመንኩ፡ ወተወከልኩ። ለዓለመ፡ ዓለም፡ አሜን፡ ተሙስ፡ ፱ጾመ፡ አብ፡ ፲ወ፯፡ ዕለተ፡ ጾም፡ ጠብፕ፡ ፫ጾመ፡ ኅዳር፡ ፲ጾም።</explicit>
+                            <listBibl type="text">
+                                <bibl>
+                                    <ptr target="bm:Aescoly1951Recueil"/>
+                                    <citedRange unit="page">186-187</citedRange>
+                                </bibl>
+                            </listBibl>
+                        </msItem>
+                        <msItem xml:id="ms_i7">
+                            <locus from="10va" to="11va"/>
+                            <title type="incomplete" corresp="LIT7504PrayerBE"/>
```

Comment:
If the text is incomplete, could you include a note with a statement on the incompleteness? Here and elsewhere

Reply:
I will create textparts and refer to text parts here and elsewhere.